docs/visualization/filters.md

```diff
@@ -0,0 +1,35 @@
+# Filters
+
+Filters is a multi-bucket aggregation where each bucket contains the documents that match a query.
+
+**Important note:** K2Bridge only supports the Lucene syntax for queries. Make sure you change the default syntax in the Filters aggregation ("KQL) to "Lucene", using the button next to the search box.
+
+![](images/filters.png)
```

Comment:
image link broken

Reply:
Not sure what the problem is? The image is there and looks OK when I open the page in GitHub